daffodil-runtime1/src/main/scala/org/apache/daffodil/processors/parsers/SpecifiedLengthParsers.scala

```diff
@@ -132,6 +133,18 @@ class SpecifiedLengthExplicitParser(
   }
 }
 
+class SpecifiedLengthChoiceParser(
+  eParser: Parser,
+  erd: RuntimeData,
+  choiceLength: JLong,
+  toBits: Int)
+  extends SpecifiedLengthParserBase(eParser, erd) {
```

Comment:
Remove toBits and change choiceLength to chocieLengthInBits, or is toBits needed by SpecifiedLengthParserBase?